ios/LinkDemo-ObjC/LinkDemo-ObjC/ViewController.m

```diff
@@ -63,6 +63,7 @@ - (void)didReceiveNotification:(NSNotification*)notification {
 - (IBAction)didTapButton:(id)sender {
 #if USE_CUSTOM_CONFIG
     [self presentPlaidLinkWithCustomConfiguration];
+    // [self presentPlaidLinkWithCopyCustomization];
```

Comment:
No, this way people can easily try out instance configuration based customization.